### ISSUE TITLE
update bower to allow angular 1.6

### ISSUE THREAD

**Issue by xuaninbox** [brantwills/Angular-Paging#61]
bower.json should be updated to allow angular 1.6

**Comment by phackwer** [comment]
Yes, it works on 1.6.*. It's installed anyway, but having this on the output... Really pisses a devops up.


Please note that,
    angular-paging#2.2.2 depends on angular#>=1.3.1 <1.6 which resolved to angular#1.5.11
    angular-animate#1.6.1, angular-cookies#1.6.1, angular-mocks#1.6.1, angular-resource#1.6.1, angular-sanitize#1.6.1, cr-acl#223c35c7c3, xrs-ux depends on angular#1.6.1 which resolved to angular#1.6.1
    angular-locker#2.0.4 depends on angular#>=1.2.0 <2.0.0 which resolved to angular#1.6.1
    oclazyload#1.0.9 depends on angular#>=1.2.0 which resolved to angular#1.6.1
    angular-ui-router#0.3.2 depends on angular#^1.0.8 which resolved to angular#1.6.1
    angucomplete-alt#3.0.0 depends on angular#>=1.6.0 which resolved to angular#1.6.1
    angular-strap#2.3.12 depends on angular#^1.2.21 which resolved to angular#1.6.1
    angular-datatables#0.5.6 depends on angular#>=1.3.0 which resolved to angular#1.6.1
Resort to using angular#1.6.1 which resolved to angular#1.6.1
Code incompatibilities may occur.

**Comment by phackwer** [comment]
DAMN!!! THIS TICKET IS AN YEAR OLD?!??! Come on? Is there anybody giving maintainance?
